config/README.rst

```diff
@@ -162,6 +169,9 @@ fail-under
 with-pypy
   Does the package support PyPy: True/False
 
+without-py2
+  Do not run the tests on Python 2, PyPy2 and Python 3.5: True/False
+
```

Comment:
Double negatives are hard to parse.  Could we have `with-py2` in the config file, defaulting to True, but keep the `--without-py2` command-line option?

Reply:
I changed the command line option to `--without-legacy-python`. In `.meta.cfg` the parameter is now called `with-legacy-python`. The default is to keep supporting Python 2.7+.